crates/common/validator/src/validator.rs

```diff
@@ -310,6 +312,35 @@ impl ValidatorService {
             .await?)
     }
 
+    pub async fn make_attestation(
+        &self,
+        slot: u64,
+        validator_index: u64,
+        committee_index: u64,
+    ) -> anyhow::Result<()> {
+        if let Some(keystore) = self.validator_index_to_keystore.get(&validator_index) {
+            let attestation_data = self
+                .beacon_api_client
+                .get_attestation_data(slot, committee_index)
+                .await?
+                .data;
+
+            Ok(self
+                .beacon_api_client
+                .submit_attestation(vec![SingleAttestation {
+                    attester_index: validator_index,
+                    committee_index,
+                    signature: sign_attestation_data(&attestation_data, &keystore.private_key)?,
+                    data: attestation_data,
+                }])
+                .await?)
+        } else {
+            Err(anyhow!(
+                "Keystore not found for validator: {validator_index}"
+            ))
+        }
+    }
```

Comment:
```suggestion
    pub async fn make_attestation(
        &self,
        slot: u64,
        validator_index: u64,
        committee_index: u64,
    ) -> anyhow::Result<()> {
        let Some(keystore) = self.validator_index_to_keystore.get(&validator_index) else {
            bail!("Keystore not found for validator: {validator_index}");
        }
   
        let attestation_data = self
            .beacon_api_client
            .get_attestation_data(slot, committee_index)
            .await?
            .data;

        Ok(self
            .beacon_api_client
            .submit_attestation(vec![SingleAttestation {
                attester_index: validator_index,
                committee_index,
                signature: sign_attestation_data(&attestation_data, &keystore.private_key)?,
                data: attestation_data,
            }])
            .await?)
    }
```

In the future if let Some should normally changed into let Some like I showed above